workers/dotnet/Temporalio.Omes.csproj

```diff
@@ -10,6 +10,10 @@
         <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     </PropertyGroup>
 
+    <!-- When testing/linting, treat this as a library so no Main method is needed. -->
+    <PropertyGroup Condition="'$(Configuration)' == 'Library'">
+        <OutputType>Library</OutputType>
+    </PropertyGroup>
```

Comment:
So this is needed because without it, the build will complain that there's no `Main`. But if you define a `Main`, the image build process will complain that there are two `Main`s ...